src/styles/components/dialog-screen.styl

```diff
@@ -2,7 +2,7 @@
   position: fixed
   left: 0
   top: 0
-  z-index: 1000000  //Place waaay above map plugin elements.
+  z-index: 3000000  //Place waaay above map plugin elements.
```

Comment:
I absolutely agree with the idea of having 'rational' z-index values. I'd like to keep the z-index low with each layer at a multiple of 10 or 100, but as I discovered...

![screen shot 2016-05-31 at 17 03 36](https://cloud.githubusercontent.com/assets/13952701/15681598/b926be6a-2751-11e6-96c0-706540d02074.png)

...the CartoDB logo itself comes in at a staggering **1-million** z-index.

![Dr Evil's One Million Dollars meme](http://www.reactiongifs.com/r/1m.gif)

Admittedly, my 3-million z-index was an overreaction - `z-index: 1,000,001 would be more than sufficient at handling the layering.